public/v4/apps/monica.yml

```diff
@@ -0,0 +1,60 @@
+captainVersion: 4
+services:
+    # Monica
+    $$cap_appname:
+        depends_on:
+            - $$cap_appname-mariadb
+        image: monica:apache
```

Comment:
I am assuming they don't have versioned tags, right?

If that's the case, I suggest we use sha hash to ensure that the one-click app won't break if they change the image and change the requirement in the future.